rotkehlchen/chain/ethereum/modules/curve/pools_cache.py

```diff
@@ -113,6 +115,8 @@ def ensure_curve_tokens_existence(
                         weight=ONE,
                     )],
                     seen=TokenSeenAt(description='Querying curve pools'),
+                    name=ethaddress_to_identifier(token_address),
+                    symbol=DEFAULT_MISSING_SYMBOL,
```

Comment:
Have you checked that the symbol doesn't end up being `DEFAULT_MISSING_SYMBOL` in all the tokens? At least by looking at this line of the code https://github.com/rotki/rotki/blob/8ae08751302aeebb844de45b28b2bbd713d8079b/rotkehlchen/assets/utils.py#L59 it sounds to me that it will be preferred over the information on chain

Reply:
Good catch. Indeed that is what would happend. I've changed to catch the error instead and skip